stable/ambassador/templates/admin-service.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ include "ambassador.fullname" . }}-admins
+  name: {{ include "ambassador.fullname" . }}-admin
```

Comment:
I would consider this a breaking change, been wanting to correct this since I realized I made the typo :D

Reply:
We don't have to include this in the PR if it would be the driving force for bumping the major version. I just ran into it earlier and spent way too long before realizing the typo :). 

If we already have to bump the major version for the `service_port` change, I say we include this fix then as well.